src/renderer/managers/plugins.ts

```diff
@@ -41,9 +41,10 @@ export function getExports(id: string): PluginExports | undefined {
 }
 
 function register(plugin: RepluggedPlugin): void {
+  const existingExports = plugins.get(plugin.manifest.id)?.exports;
```

Comment:
again, `getExports` does exist lolll

Reply:
`getExports` will throw if the plugin doesn't exist. We don't want that.